pkg/bootstrap/render.go

```diff
@@ -46,6 +46,31 @@ var additionalClusterRoleFiles = []string{
 	"manifests/klusterlet/clusterrole_aggregate.yaml",
 }
 
+var reservedClusterClaims = []string{
+	"above.threshold.hostedclustercount.hypershift.openshift.io",
+	"apiserverurl.openshift.io",
+	"consoleurl.cluster.open-cluster-management.io",
+	"controlplanetopology.openshift.io",
+	"exclude-from-backup.velero.io",
+	"full.hostedclustercount.hypershift.openshift.io",
+	"hostingcluster.hypershift.openshift.io",
+	"id.k8s.io",
+	"id.openshift.io",
+	"infrastructure.openshift.io",
+	"kubeversion.open-cluster-management.io",
+	"local-cluster",
+	"name",
+	"nodearchitecture",
+	"oauthredirecturis.openshift.io",
+	"openshiftversion-major",
+	"openshiftversion-major-minor",
+	"platform.open-cluster-management.io",
+	"product.open-cluster-management.io",
+	"schedulable.open-cluster-management.io",
+	"version.openshift.io",
+	"zero.hostedclustercount.hypershift.openshift.io",
+}
+
```

Comment:
It's unnecessary to list all of them. Just include the following suffixes:
- "hypershift.openshift.io"
- "open-cluster-management.io"
- "openshift.io"

Reply:
There are 2 claims that I am not sure whether suffix is better:
```
	"exclude-from-backup.velero.io",
	"id.k8s.io",
```
or

```
	"velero.io",
	"k8s.io",
```

I don't know if customers need to use the `velero.io` and `k8s.io` as their claims' domain suffix.